lib/awful/hotkeys_popup/widget.lua

```diff
@@ -44,22 +55,71 @@ widget.hide_without_description = true
 widget.merge_duplicates = true
 
 
+--- Widget background color.
+-- @beautiful beautiful.hotkeys_bg
```

Comment:
I really feel like I am nitpicking right now, but you should use the "hotkeys" or "hotkeys widget" in the description and add the type to all of them. See https://awesomewm.org/apidoc/documentation/06-appearance.md.html 

Not really important, but coherency, boring predictability and conventions goes a long way when you document a library.